mrmustard/lab_dev/wires.py

```diff
@@ -325,6 +325,11 @@ def indices(self) -> tuple[int, ...]:
             self.index_dicts[t][m] for t, modes in enumerate(self.sorted_args) for m in modes
         )
 
+    
+    @cached_property
```

Comment:
Why are we caching this? This seems unnecessary as it's just a reordering of an already cached property i.e. duplicate information being cached